src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net8.Manifest/WorkloadManifest.json.in

```diff
@@ -18,8 +18,7 @@
       "description": ".NET WebAssembly experimental tooling for net8.0",
       "packs": [
         "Microsoft.NET.Runtime.WebAssembly.Templates.net8",
-        "Microsoft.NETCore.App.Runtime.Mono.multithread.net8.browser-wasm",
-        "Microsoft.NETCore.App.Runtime.Mono.perftrace.net8.browser-wasm"
+        "Microsoft.NETCore.App.Runtime.Mono.multithread.net8.browser-wasm"
```

Comment:
just to be clear the workload manifest parse explicitly allows these

Reply:
What do you mean?

Reply:
just that the trailing ' is fine

Reply:
> just that the trailing ' is fine

ah:)